UnitsNet/CustomCode/UnitValueAbbreviationLookup.cs

```diff
@@ -63,10 +63,9 @@ internal List<string> GetAbbreviationsForUnit(int unit)
 
         internal List<int> GetUnitsForAbbreviation(string abbreviation)
         {
-            if(!abbreviationToUnitMap.TryGetValue(abbreviation, out var units))
-                abbreviationToUnitMap[abbreviation] = units = new List<int>();
-
-            return units.Distinct().ToList();
+            return abbreviationToUnitMap
+                .Where(x => x.Key.Equals(abbreviation, StringComparison.OrdinalIgnoreCase))
```

Comment:
I'm a bit on the fence on whether to use OrdinalIgnoreCase or InvariantCultureIgnoreCase.
https://stackoverflow.com/a/72766/134761

I guess since we are not parsing random user input, but our own type names and unit abbreviations, it's perhaps fine have exact matching (ordinal) instead of culture-specific matching. It is faster too.

Reply:
I originally wanted to use InvariantCultureIgnoreCase, but it wouldn't build. Because we are defining our abbreviations statically, either string comparison methods will work.